.storybook/stories/select/select.stories.ts

```diff
@@ -37,7 +37,9 @@ const selectTemplate: StoryFn = args => ({
     <clr-select-container>
       <label>Options</label>
       <select clrSelect>
-        <option *ngFor="let _ of createArray(optionCount); let i = index" [value]="i + 1">Option {{ i + 1 }}</option>
+        <option *ngFor="let _ of createArray(optionCount); let i = index" [value]="i + 1" [disabled]="i === 1">
```

Comment:
```suggestion
        <option *ngFor="let _ of createArray(optionCount); let i = index" [value]="i + 1" [disabled]="i === 0">
```

Should this one be consistent with the multiple select story?

Reply:
Doesn't really matter.

I changed the single selection story. 
Commit e8fb100